sensor/scripts/run_all_tests.sh

```diff
@@ -0,0 +1,2 @@
+pytest --cov=src --cov=cli tests
+exit $?
```

Comment:
That's slightly shady 😉 Have a look at [this nice article](https://sharats.me/posts/shell-script-best-practices/), I don't agree with number 3, but most of the points are pretty good.

Particularly use Number 4 (`set -o errexit`) instead of your duct tape solution.

Same thing for the other shell scripts.

Reply:
Hahaha. But my version is shorter ;)